src/coord/src/catalog.rs

```diff
@@ -210,6 +230,10 @@ impl CatalogState {
         &self.by_id[id]
     }
 
+    pub fn entries(&self) -> impl Iterator<Item = &CatalogEntry> {
+        self.by_id.values()
+    }
```

Comment:
This looks like dead code now to me. (If we do keep it, then we should re-implement `Catalog::entries` in terms of this function.)